vint.go

```diff
@@ -60,3 +60,37 @@ func mustReadByte(r io.Reader) (byte, error) {
 
 	return b, err
 }
+
+// WriteVInt writes an int64 encoded in hadoop's "VInt" format, described and
+// implemented here: https://goo.gl/1h4mrG. It does at most one write to the
+// underlying io.Writer
```

Comment:
also I'd say "It does a single write to the underlying io.Writer, and returns the number of bytes written."